crates/bevy_window/src/window.rs

```diff
@@ -277,6 +277,15 @@ pub struct Window {
     ///
     /// - Only supported on Windows.
     pub skip_taskbar: bool,
+    /// Sets whether the window should clip its child windows to its client area.
+    ///
+    /// If `true`, child windows will be clipped to the client area of this window.
+    /// If `false`, child windows can be drawn outside the client area.
```

Comment:
This wording here isn't entirely clear to me but maybe that's just because I'm familiar with Windows terminology. I have a suggestion but it's not blocking.

```suggestion
    /// Sets whether the window should draw over its child windows.
    ///
    /// If `true`, the window excludes drawing over areas obscured by child windows.
    /// If `false`, the window can draw over child windows.
```

Reply:
Your explanation seems more concise than mine but same as you, I was having trouble with with the wording.
I'll go ahead and accept the changes.